src/Command/InfectionCommand.php

```diff
@@ -103,6 +104,13 @@ protected function configure()
                 InputOption::VALUE_REQUIRED,
                 'Custom configuration file path'
             )
+            ->addOption(
+                'coverage',
+                null,
+                InputOption::VALUE_REQUIRED,
```

Comment:
I'd say that this option shouldn't be mandatory. Can we do it optional?

Reply:
This parameter means that the value of the option is required, not for the option itself.

So `--coverage=dir` is allowed, but `--coverage` is not